extensions/deepcast/src/utils.ts

```diff
@@ -5,21 +5,45 @@ interface Preferences {
   key: string;
   pro: boolean;
 }
-const translate = async (target: string) => {
-  const preferences: Preferences = getPreferenceValues();
-  const key = preferences.key;
-  const pro = preferences.pro;
+
+export function getPreferences() {
```

Comment:
Where is the point of defining it in an extra function when you use it only once?

Reply:
Absolutely valid claim. My idea was to keep a unified function which would provide typed preferences, in case some other parts of deepcast would need to access them.

Since only 1 function utilizes preferences now, it may be redundant to keep a dedicated function for retrieving them. Up to you, whether to keep `getPreferences` or remove it.

Reply:
Makes sense.